src/api/java/com/github/minecraftschurlimods/arsmagicalegacy/api/ArsMagicaAPI.java

```diff
@@ -50,162 +47,128 @@ private ArsMagicaAPI() {
-         *
-         * @return the {@link IAffinityHelper} instance
+         * @return The affinity helper instance.
          */
         @Unmodifiable
         IAffinityHelper getAffinityHelper();
 
         /**
-         * Get the {@link IMagicHelper} instance.
-         *
-         * @return the {@link IMagicHelper} instance
+         * @return The magic helper instance.
          */
         @Unmodifiable
         IMagicHelper getMagicHelper();
 
         /**
-         * Get the {@link IManaHelper} instance.
-         *
-         * @return the {@link IManaHelper} instance
+         * @return The mana helper instance.
          */
         @Unmodifiable
         IManaHelper getManaHelper();
 
         /**
-         * Get the {@link IBurnoutHelper} instance.
-         *
-         * @return the {@link IBurnoutHelper} instance
+         * @return The burnout helper instance.
          */
         @Unmodifiable
         IBurnoutHelper getBurnoutHelper();
 
         /**
-         * Get the {@link ISpellHelper} instance.
-         *
-         * @return the {@link ISpellHelper} instance
+         * @return The spell helper instance.
          */
         @Unmodifiable
         ISpellHelper getSpellHelper();
 
         /**
-         * Get the {@link IRiftHelper} instance.
-         *
-         * @return the {@link IRiftHelper} instance
+         * @return The rift helper instance.
          */
         @Unmodifiable
         IRiftHelper getRiftHelper();
 
         /**
-         * Get the {@link IShrinkHelper} instance.
-         *
-         * @return the {@link IShrinkHelper} instance
+         * @return The shrink helper instance.
          */
         @Unmodifiable
         IShrinkHelper getShrinkHelper();
 
         /**
-         * Get the {@link IEtheriumHelper} instance.
-         *
-         * @return the {@link IEtheriumHelper} instance
+         * @return The etherium helper instance.
          */
         IEtheriumHelper getEtheriumHelper();
 
         /**
-         * Open the occulus gui for the given player.
+         * Opens the occulus gui for the given player.
          *
-         * @param player the player to open the gui for
+         * @param player The player to open the gui for.
          */
         void openOcculusGui(Player player);
 
         /**
-         * Open the spell customization gui for the given spell (the given stack).<br>
-         * Only works on the client.
+         * Opens the spell customization gui for the given player.
+         *
+         * @param level  The level to open the gui in.
+         * @param player The player to open the gui for.
+         * @param stack  The spell item stack to open the gui for.
          */
         void openSpellCustomizationGui(Level level, Player player, ItemStack stack);
     }
 
-    @SuppressWarnings("ConstantConditions")
-    private static final class StubArsMagicaAPI implements IArsMagicaAPI {
+    private static class StubArsMagicaAPI implements IArsMagicaAPI {
```

Comment:
Why remove the final and the warning suppression?

Reply:
final is useless with private (IntelliJ suggestion)
I removed @SuppressWarnings everywhere

Reply:
> I removed @SuppressWarnings everywhere

Leave them in